app/controllers/frequently_asked_questions_controller.rb

```diff
@@ -1,44 +1,56 @@
 class FrequentlyAskedQuestionsController < ApplicationController
-  load_and_authorize_resource
+  # load_and_authorize_resource
```

Comment:
@NRaeewal do you want to keep this comment?

Reply:
Nope! Fixed